tests/cli/commands/test_connection_command.py

```diff
@@ -321,3 +330,87 @@ def test_cli_delete_invalid_connection(self):
 
         # Check deletion attempt stdout
         self.assertIn("\tDid not find a connection with `conn_id`=fake", stdout)
+
+
+class TestCliImportConnections(unittest.TestCase):
```

Comment:
It would be valuable to test also for emergency situations when an operation fails, or the user gives an incorrect value as a parameter.

Reply:
@mik-laj,  I can think of a few scenarios which are already handled:

> 1. Missing file scenario is handled by cli_parser.py
> 2. Unsupported file format is handled in local_filesystems.py
> 3. Parsing error is handled in local_filesystems.py
> 4. Wrong parameter for --conflict-disposition is handled by cli_parser.py

Can you provide an example?